trackstop.lua

```diff
@@ -236,7 +268,197 @@ function RollerOverlay:init()
   }
 end
 
+ReorderStopsWindow = defclass(ReorderStopsWindow, widgets.Window)
+ReorderStopsWindow.ATTRS {
+  frame={t=4,l=60,w=49, h=26},
+  frame_title='Reorder Stops',
+  resizable=true,
+}
+
+local SELECT_STOP_HINT = 'Select a stop to move'
+local SELECT_ANOTHER_STOP_HINT = 'Select another stop to swap or same to cancel'
```

Comment:
right click should also cancel out of a partially completed reorder operation. you can do this by handling `keys._MOUSE_R` in `ReorderStopsWindow:onInput(keys)` and returning `true` if you have handled the right click and don't want the window to close.

Reply:
Fixed